docs/source/cloud-setup/cloud-permissions/gcp.rst

```diff
@@ -152,6 +151,12 @@ User
     compute.images.get
     compute.images.useReadOnly
 
+9. **Optional**: If the user sets ``gcp.prioritize_reservations`` or ``gcp.specific_reservations`` in :ref:`~/.sky/config.yaml <config-yaml>`, you can additionally add the following permissions:
+
+.. code-block:: text
+
+    compute.reservations.list
```

Comment:
Do we need to add `get` as well? Used in code.

Reply:
Tried without `get` permisison, and it seems instance creation with `specific_reservation` still works : )

Reply:
It seems the `get` permission is not required for listing and creating the instance with the reservation. Just removed it from code.